npm/webpack-dev-server/test/e2e.spec.ts

```diff
@@ -87,9 +101,13 @@ describe('#startDevServer', () => {
       },
     })
 
-    return new Promise((res) => {
-      devServerEvents.on('dev-server:compile:success', () => {
-        close(() => res())
+    return new Promise<void>((res) => {
```

Comment:
What does this change do? I don't understand the purpose/what this is testing.

Reply:
If I'm understanding correctly before this change the `dev-server:compile:success` event would fire after compiling and outputting the compiled files. So this test would make sure after the dev server was done starting up we got that event. 

With the proposed change it is now firing after a successful rebuild and if the changed file is one of the specs we're watching. So before we can run the old test we want to modify a file to trigger a rebuild. 

If that's a correct description of what we're doing, do we want a test where we test touching a file that isn't being watched?